annotator-core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -263,6 +263,22 @@ private void forceResolveRemainingErrors() {
     // Update log.
     config.log.updateInjectedAnnotations(nullUnMarkedAnnotations);
 
+    // Collect @NullUnmarked annotations for classes for errors in static initialization blocks.
```

Comment:
What about instance initialization blocks?  E.g., you could have:

```java
class Foo {

  {
    getNullable().toString();
  }

  @Nullable static Object getNullable() { return null; }
}
```

Will our logic handle this case too?

Reply:
Yes it does 🙂. Updated a unit test to verify this 47998c01f5b1f7148818a15f711ba424ae84b3d9

Reply:
Maybe we should do some quick renaming in comments / code where you refer to static blocks, to call them initializer blocks (or just `INIT_BLOCK` to make it shorter)?

Reply:
Changed to `INIT_BLOCK` cafef181d7e0027c93a3d7a525cf21f5bbd62344